Install-DVWA.sh

```diff
@@ -0,0 +1,198 @@
+⣿⣿⣿⣿⣿⣿⣿⣿⣿⣷⣥⣀⡁⠄⠘⠘⠘⢀⣠⣾⣿⢿⣦⣁⠙⠃⠄⠃⠐⣀
+⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣋⣵⣾⣿⣿⣿⣿⣦⣀⣶⣾⣿⣿⡉⠉⠉
+⣿⣿⣿⣿⣿⣿⣿⠟⣫⣥⣬⣭⣛⠿⢿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡆⠄
+⣿⣿⣿⣿⣿⣿⣿⠸⣿⣏⣙⠿⣿⣿⣶⣦⣍⣙⠿⠿⠿⠿⠿⠿⠿⠿⣛⣩⣶⠄
+⣛⣛⣛⠿⠿⣿⣿⣿⣮⣙⠿⢿⣶⣶⣭⣭⣛⣛⣛⣛⠛⠛⠻⣛⣛⣛⣛⣋⠁⢀
+⣿⣿⣿⣿⣿⣶⣬⢙⡻⠿⠿⣷⣤⣝⣛⣛⣛⣛⣛⣛⣛⣛⠛⠛⣛⣛⠛⣡⣴⣿
+⣛⣛⠛⠛⠛⣛⡑⡿⢻⢻⠲⢆⢹⣿⣿⣿⣿⣿⣿⠿⠿⠟⡴⢻⢋⠻⣟⠈⠿⠿
+⣿⡿⡿⣿⢷⢤⠄⡔⡘⣃⢃⢰⡦⡤⡤⢤⢤⢤⠒⠞⠳⢸⠃⡆⢸⠄⠟⠸⠛⢿
+⡟⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠁⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⠄⢸
+\033[0m"
+
+echo -e "\033[96mWelcome to the DVWA installer!\033[0m"
+
+echo
+echo
+# Function to display instructions in English
+show_english_instructions() {
+	echo -e "\e[96mUpdating repositories...\e[0m"
+	apt update > /dev/null 2>&1
+	clear
+    
+	echo -e "\e[96mVerifying and installing necessary dependencies...\e[0m"
+    
+    check_program apache2
+    check_program mariadb-server
+    check_program mariadb-client
+    check_program php
+    check_program php-mysqli
+    check_program php-gd
+    check_program libapache2-mod-php
+    check_program git
+    sleep 2
+	clear
+    
+	echo -e "\e[96mDownloading DVWA from GitHub...\e[0m"
+    git clone https://github.com/ethicalhack3r/DVWA.git /var/www/html/DVWA
+	sleep 2
+	clear
+    
+	echo -e "\e[96mStarting MySQL...\e[0m"
+    systemctl start mysql.service
+	sleep 2
+	clear
+    
+	echo -e "\e[96mConfiguring the database for DVWA...\e[0m"
+    mysql -u root -e "CREATE DATABASE IF NOT EXISTS dvwa;"
+    mysql -u root -e "CREATE USER 'dvwa'@'localhost' IDENTIFIED BY 'abc123';"
+    mysql -u root -e "GRANT ALL PRIVILEGES ON dvwa.* TO 'dvwa'@'localhost';"
+    mysql -u root -e "FLUSH PRIVILEGES;"
+	sleep 2
+	
+    echo -e "\e[96mConfiguring DVWA...\e[0m"
+    cp /var/www/html/DVWA/config/config.inc.php.dist /var/www/html/DVWA/config/config.inc.php
+    sed -i "s/\(\$_DVWA\[ 'db_password' \] = '\).*\('\)/\1abc123\2/" /var/www/html/DVWA/config/config.inc.php
+    sleep 2
+
+    echo -e "\e[96mConfiguring permissions...\e[0m"
+    chown -R www-data:www-data /var/www/html/DVWA
+    chmod -R 755 /var/www/html/DVWA
+    sleep 2
+
+    echo -e "\e[96mConfiguring PHP...\e[0m"
+    php_config_file="/etc/php/$(php -r 'echo PHP_MAJOR_VERSION . "." . PHP_MINOR_VERSION;')/apache2/php.ini"
+    sed -i 's/^\(allow_url_include =\).*/\1 on/' $php_config_file
+    sed -i 's/^\(allow_url_fopen =\).*/\1 on/' $php_config_file
+    sed -i 's/^\(display_errors =\).*/\1 on/' $php_config_file
+    sed -i 's/^\(display_startup_errors =\).*/\1 on/' $php_config_file
+    sleep 2
+
+    echo -e "\e[96mRestarting Apache...\e[0m"
+    systemctl restart apache2
+    sleep 2
+
+    echo -e "\e[92mUsername and password for the first use:\e[0m"
+	echo -e "Username: \033[93mdvwa\033[0m"
+	echo -e "Password: \033[93mabc123\033[0m"
+
+    echo -e "\e[92mDVWA has been installed successfully. Access \e[93mhttp://localhost/DVWA\e[0m to get started."
+
+    echo -e "\e[92mCredentials after setup:\e[0m"
+	echo -e "Username: \033[93madmin\033[0m"
+	echo -e "Password: \033[93mpassword\033[0m"
+  echo
+  echo
+  echo -e "\033[91mWith ♡ by Iamcarron"
+}
+# Function to display instructions in Spanish
+show_spanish_instructions() {
```

Comment:
I like having the option, but don't like having all the repeated code, it makes it too easy for one to get out of sync. 

It would be a mess to have if/switch statements on every output string so could you look if there is a recommended way to do localisation in scripts?